### DISCUSSION TITLE
(outdated)

### DISCUSSION THREAD

**Discussion by 8bitmp3** [jax-ml/jax#3625] [General] [answered]
Live stream at 11AM Pacific Time here https://www.youtube.com/watch?v=efcRhdA4Ql0

https://twimlai.com/the-great-ml-language-un-debate/
> We’re excited to bring you The Great ML Language (Un)Debate, another in our series of live discussion sessions. Join us for a thoughtful discussion exploring the strengths, weaknesses, and approaches of both popular and emerging programming languages for machine learning like Python, R, C++, Swift, Julia, Clojure, and more. This session will also include an opportunity for you to engage with experts and peers on the topic.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/19637339/86265033-32931580-bbbb-11ea-909b-e1edfb41077c.png">

cc @hawkinsp @skye @mattjj Python ✅ but no C++ representation in the panel

**Comment by shoyer** [comment, marked as answer]
Providing an "answer" to indicate that this "discussion" is closed...
